core/src/test/java/io/substrait/type/proto/LocalFilesRoundtripTest.java

```diff
@@ -0,0 +1,114 @@
+package io.substrait.type.proto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.substrait.expression.ExpressionCreator;
+import io.substrait.expression.FieldReference;
+import io.substrait.expression.ImmutableFieldReference;
+import io.substrait.expression.proto.FunctionCollector;
+import io.substrait.function.SimpleExtension;
+import io.substrait.proto.ReadRel;
+import io.substrait.relation.LocalFiles;
+import io.substrait.relation.ProtoRelConverter;
+import io.substrait.relation.RelProtoConverter;
+import io.substrait.relation.files.FileOrFiles;
+import io.substrait.relation.files.ImmutableFileFormat;
+import io.substrait.relation.files.ImmutableFileOrFiles;
+import io.substrait.type.ImmutableNamedStruct;
+import io.substrait.type.Type;
+import io.substrait.type.TypeCreator;
+import java.io.IOException;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+public class LocalFilesRoundtripTest {
```

Comment:
@jacques-n 

Do you mean this?

Reply:
Yes, apparently missed that on first pass. Thanks